### PR TITLE
Bug/GfCarouselComponent Fixed

### PR DESCRIPTION
**Pull request by dinirichard**

### CODE REVIEW

File: libs/ui/src/lib/carousel/carousel.component.ts [hunk 1]
```diff
@@ -30,6 +31,7 @@ import { CarouselItem } from './carousel-item.directive';
 })
 export class GfCarouselComponent implements AfterContentInit {
   @ContentChildren(CarouselItem) public items!: QueryList<CarouselItem>;
+  itemsSignal = contentChildren('carouselItem', { read: ElementRef });
```
Comment (dtslvr) [inline libs/ui/src/lib/carousel/carousel.component.ts:34]: Do we still need both `items` and `itemsSignal`?

Comment (dtslvr) [inline libs/ui/src/lib/carousel/carousel.component.ts:34]: `private itemsSignal = ...` ❓

File: libs/ui/src/lib/carousel/carousel.component.ts [hunk 2]
```diff
@@ -120,16 +121,15 @@ export class GfCarouselComponent implements AfterContentInit {
       return;
     }
 
-    const itemsArray = this.items.toArray();
+    const itemsArray = this.itemsSignal();
```
Comment (dtslvr) [inline libs/ui/src/lib/carousel/carousel.component.ts:124]: `const carouselItemRefs = this.itemsSignal();` ❓

**Review by dtslvr — CHANGES_REQUESTED** [review]
Thanks for your contribution, @dinirichard! 🙂
I have successfully tested it and added a few comments.

File: libs/ui/src/lib/carousel/carousel.component.ts (hunk 1 above)
Comment (dinirichard) [reply libs/ui/src/lib/carousel/carousel.component.ts:34]: Is Keyboard focus and navigation `onKeyDown()` necessary?

Comment (dtslvr) [reply libs/ui/src/lib/carousel/carousel.component.ts:34]: We can ditch the keyboard support altogether to simplify this component.

**Review by dtslvr — APPROVED** [review]
Thanks @dinirichard! 👍🏻
Ready to merge now.

**Pull request merged**